enterprise-suite/values.yaml

```diff
@@ -2,9 +2,9 @@
 # Update these versions for a release.
 #
 # lightbend-docker-commercial-registry.bintray.io/enterprise-suite/es-console
-esConsoleVersion: v0.1.7
+esConsoleVersion: rc
 # lightbend-docker-commercial-registry.bintray.io/enterprise-suite/es-monitor-api
-esMonitorVersion: v0.2.1
+esMonitorVersion: rc
 # lightbend-docker-commercial-registry.bintray.io/enterprise-suite/es-grafana
 esGrafanaVersion: v0.1.4
 # prom/prometheus
```

Comment:
I'm guessing we'll change these versions before the PR is merged, right?

Reply:
this is just temporary staging, to be thrown away once we merge to master